### PR TITLE
feat(front): add option to easily change lang

### LINKED ISSUE #135: Possibilité de changer le langage du bin par l'UI.
On pourrait ajouter la fonctionnalité de changer le langage du bin directement via une liste comme pour créer un bin, cela pourrait améliorer l'utilisation sur les mobiles, car quand on ouvre un bin sur téléphone celui si vas s'ouvrir dans le navigateur interne de Discord ce qui nous offre pas la possibilité de modifier l'url la seule façon à ce moment là c'est d'ouvrir, le bin dans le navigateur externe, et cela permettrais d'éviter que l'utilisateur met un langage non supporté par le serveur.

### PR DESCRIPTION
**Pull request by Mesteery**
Closes: https://github.com/readthedocs-fr/bin-server/issues/135

### CODE REVIEW

File: bin/views/highlight.html [hunk 1]
```diff
@@ -6,6 +6,14 @@
...
         <script src="/assets/scripts/loc.js" defer></script>
         <script src="/assets/scripts/report.js" defer></script>
+        <script src="/assets/scripts/change-lang.js" defer></script>
+        <noscript>
+            <style>
+                #lang-selection {
+                    display: none;
+                }
+            </style>
+        </noscript>
```
Comment (Julien00859) [inline bin/views/highlight.html:16]: Maybe we can rename `report.css` to `hightlight.css` and store that css bit in that renamed file ?

File: bin/views/highlight.html [hunk 2]
```diff
@@ -22,6 +30,17 @@
...
         <div class="controls">
+            <select id=lang-selection class="control" title="Langage">
+                <option disabled>Langage</option>
+                % for ext_, lang_ in languages:
+                % if lang_ == lang:
+                    <option value="{{ext_}}" selected>{{lang_.title()}}</option>
+                % else:
+                    <option value="{{ext_}}">{{lang_.title()}}</option>
+                % end
+                % end
```
Comment (Julien00859) [inline bin/views/highlight.html]:
```suggestion
                % for ext_, lang_ in languages:
                    % if lang_ == lang:
                        <option value="{{ext_}}" selected>{{lang_.title()}}</option>
                    % else:
                        <option value="{{ext_}}">{{lang_.title()}}</option>
                    % end
                % end
```

**Review by Julien00859 — APPROVED** [review]

File: bin/views/highlight.html (hunk 1 above)
Comment (Mesteery) [reply bin/views/highlight.html:16]: note the `<noscript>`

Comment (Julien00859) [reply bin/views/highlight.html:16]: I don't understand, what does it do?

Comment (Mesteery) [reply bin/views/highlight.html:16]: It hide the `select` when the client has JavaScript disabled, because this feature requires JavaScript to work.

**Pull request merged**